tests/test_models/phelps_deaton.py

```diff
@@ -1,65 +1,71 @@
-"""Define example model specifications."""
+"""Example specifications of the Phelps-Deaton model."""
 
 import jax.numpy as jnp
 
-RETIREMENT_AGE = 65
-N_CHOICE_GRID_POINTS = 500
-N_STATE_GRID_POINTS = 100
+# ======================================================================================
+# Numerical parameters and constants
+# ======================================================================================
 
+N_GRID_POINTS = {
+    "states": 100,
+    "choices": 500,
+}
 
-def phelps_deaton_utility_with_shock(
-    consumption,
-    working,
-    delta,
-    additive_utility_shock,
-):
-    return jnp.log(consumption) + additive_utility_shock - delta * working
+RETIREMENT_AGE = 65
 
+# ======================================================================================
+# Model functions
+# ======================================================================================
 
-def phelps_deaton_utility(consumption, working, delta):
+
+# --------------------------------------------------------------------------------------
+# Utility functions
+# --------------------------------------------------------------------------------------
+def utility(consumption, working, delta):
     return jnp.log(consumption) - delta * working
 
 
-def phelps_deaton_utility_with_filter(
+def utility_with_filter(
```

Comment:
We need this function because ATM, we have to put all states into the utility function, right? Maybe add a to-do to schedule it for removal, which is associated with the respective issue.

Reply:
Yes, exactly! Very good point, will do that!